ynr/apps/people/migrations/0044_populate_biography_timestamp.py

```diff
@@ -0,0 +1,33 @@
+from candidates.diffs import get_version_diffs
+from django.db import migrations
+
+
+def populate_biography_last_updated_timestamp(apps, schema_editor):
+    Person = apps.get_model("people", "Person")
+    for person in Person.objects.all():
+        try:
+            diffs = get_version_diffs(person.versions)
+            if diffs:
+                for diff in diffs:
+                    if diff["data"]["biography"]:
+                        person.biography_last_updated = diff["timestamp"]
```

Comment:
Can you call [`make_aware`](https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.timezone.make_aware) on `diff["timestamp"]` please?